src/Neurthino.jl

```diff
@@ -1,7 +1,23 @@
 module Neurthino
 
-import Polynomials: Poly
+using LinearAlgebra
+using SparseArrays
+using PhysicalConstants
+using Unitful
+using Polynomials
+
+import Base
+export transprob, OscillationParameters, PMNSMatrix, Hamiltonian, MatterOscillationMatrices
+
+# TEMPORARY UNTIL PhysicalConstants.jl GETS UPDATED
```

Comment:
Yeah it is not that clean as it could be, but as I wrote in the comment this was expected to survive just a few days. Unfortunately I didn't have the time a.k.a. to finish my PR in the PhysicalConstants project.

Reply:
😉